src/utils/common.ts

```diff
@@ -14,6 +14,9 @@ export type Decimalish = string | number | Decimal;
 export const AddressZero = ethers.constants.AddressZero;
 export const MAX_BIG_INT = BigNumber.from(Number.MAX_SAFE_INTEGER.toString());
 
+// Lowest ConfigStore version where the UBA model is in effect.
+export const UBA_MIN_CONFIG_STORE_VERSION = 2;
```

Comment:
We may want to comment this wish `/* */` so that IDE's like VSCode can get context
```suggestion
/** Lowest ConfigStore version where the UBA model is in effect. */
export const UBA_MIN_CONFIG_STORE_VERSION = 2;
```

Reply:
`/* */` is actually my preferred style, but I recall getting some feedback early on that the internal standard is to use `//` except for when the comment has special formatting. @mrice32 wdyt?

Reply:
Sorry, I'm confused, why are /* comments more parseable than //?

Reply:
I'm just referring to valid JSDoc formatting. This formatting is interpreted by VSCode and several other IDEs to provide additional context about functions & variables.

https://jsdoc.app/about-getting-started.html